docs/cloud/cloud-tables/cloud-table-upload-data.md

```diff
@@ -0,0 +1,61 @@
+---
+title: Upload data
+layout: default
+parent: Manage tables
+grand_parent: Cloud
+nav_order: 2
+---
+
+# How do I upload data to FeatureBase Cloud?
+{: .no_toc }
+
+
+## Before you begin
+
+{% include /cloud/cloud-before-begin.md %}
+* [Create a database](/docs/cloud/cloud-databases/cloud-db-create)
+
+## Naming standards
+
+{% include /concepts/standard-naming-obj.md%}
+{% include /cloud-table/cloud-standard-naming-table.md %}
+
```

Comment:
Add CSV structure heading here and pull structural information from the two NOTE callout boxes as directed below.